### DISCUSSION TITLE
2 * Sizif-512 builds with different results [Fixed!]

### DISCUSSION THREAD

**Discussion by rapiqui** [UzixLS/zx-sizif-512#14] [PCB Build]
Hi,

I started the journey of trying to create a couple of Sizif boards about two months ago.
Having assembled both in parallel and with the same components batch I found for my surprise that the first unit worked straightforward, whereas the second one isn't able to initialize properly and remains in this garbage screen:

![IMG_20220707_172152](https://user-images.githubusercontent.com/55879300/177885555-5895bb12-3b1b-4ba6-84f4-f72fcc5af846.jpg)

In both cases I was able to program the cpld without hassle with Quartus.

![IMG_20220620_000156](https://user-images.githubusercontent.com/55879300/177886908-771fa52a-c9bf-4a27-abf3-a3f3175833f4.jpg)

I have tried to swap CPU, SRAM, 74HCU04AP, W27C512 EPROM (enabled with jp3 in both boards) and even the AY, just in case with no results.

Measured the 28 Mhz clock and had a measure of 27.998 Mhz and even tried to replace the 1M resistor for 850k made no difference.

Measured voltage on CPU: 4.85V and on SRAM: 3.2V.

I finally tried to run Brendan Alfords test from a Dandanator and this didn't get initialized, showing the same garbage.

Resetting the Sizif makes that border could occasionally change border color (green, pink, black or even white): may this be related to memory addressing?

I was going to consider reflowing the Altera Max II, but I would prefer to go for something more consistent before trying that now, as all the components are installed and doing so is a real headhache.

Here a pic of one of the boards finished:

![IMG_20220707_171932](https://user-images.githubusercontent.com/55879300/177887043-61376d6d-f20a-4b4d-9877-8370a4aaf519.jpg)

Any help is welcome.

Cheers

**Comment by UzixLS** [comment]
I would check soldering with microscope or lens at first.
Also there is common issue with logic converters from ali - many of them are broken. To check them look to their outputs with oscilloscope.

**Comment by rapiqui** [comment]
Hi,

I would check the logic converters outputs and compare them with the
working unit.

I saw some messages regarding that in zx-pk.ru, and one of the affected users stated that bought 40 ics in the same AE shop I did and many of them
were defective...

Cheers

**Reply by rapiqui** [reply]
Hi. I finally checked some of the converter outputs and found some strange values. Below is a screenshot of U10 on pin 15:  
![SAVE_20220721_231715](https://user-images.githubusercontent.com/55879300/180402522-b349f6de-59ce-4bea-9176-5bf57d7303d5.jpg)

Whereas, on the working Sizif-512:

![SAVE_20220721_231735](https://user-images.githubusercontent.com/55879300/180402823-1d931584-e974-4d59-9127-1f476f24e253.jpg)

I replaced the ic for another of the same package, but I had similar results to the first picture.
Are both defective?

**Reply by rapiqui** [reply]
Fixed! The culprit were the bus switches. I was checking the wrong pins on the ICs. I had to check the A1/A8 lines (pins 2-9), where I discovered that U5 and U3 had no output. Changed them for other two KS245 and luckily those were OK. After the swap...the Sizif came to life.
